server/src/test/java/io/druid/segment/loading/StorageLocationTest.java

```diff
@@ -32,11 +35,15 @@
  */
 public class StorageLocationTest
 {
+  @Rule
+  public final TemporaryFolder temporaryFolder = new TemporaryFolder();
+
   @Test
   public void testStorageLocation() throws Exception
   {
     long expectedAvail = 1000L;
-    StorageLocation loc = new StorageLocation(new File("/tmp"), expectedAvail);
+    File location = temporaryFolder.newFolder("/tmp");
```

Comment:
Can this just be `newFolder()`? The `/tmp` requirement previously existing is probably a bug in the testing procedure.